build.gradle

```
@@ -88,6 +88,9 @@ dependencies {
     implementation "io.github.openfeign:feign-jackson:$openFeignVersion"
     implementation "io.github.openfeign:feign-slf4j:$openFeignVersion"
 
+    // expiring map
+    implementation "net.jodah:expiringmap:0.5.10"
```

Should we update to the last version?
```suggestion
    implementation "net.jodah:expiringmap:0.5.11"
```

---

ok, I'm going to upgrade